pxblue/angular-themes
# Add github CI workflow

Fixes BLUI-5563.



#### Changes proposed in this Pull Request:

- add github CI workflow
- remove circleCI config
- stop building in circleCI



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- make small commit to see job run and remove commit
- https://github.com/etn-ccis/blui-angular-themes/actions/runs/8484561667



#### Any specific feedback you are looking for?

-
